src/background/services/key/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface KeyPair {
+  publicKey: string;
+  secretKey: string;
```

Comment:
Should we rename this to `privateKey`?

Reply:
The same fields are in tweetnacl library.